### PR TITLE
Moving requirements.txt to environment.yml for biology issue #2934

### PR DESCRIPTION
**Pull request by felder**
Also setting timezone to America/Los_Angeles

### CODE REVIEW

**Comment by felder** [comment]
@petersudmant just giving you a heads up that as per issue #2934 I'm folding requirements.txt into environment.yml  I also set the timezone to America/Los_Angeles as we've previously discussed.

**Comment by felder** [comment]
@yuvipanda just checking in with you regarding items conda didn't know about.

https://github.com/berkeley-dsep-infra/datahub/blob/2614e021304c1cb6f0a546ec5fba7e99aef3e6b8/deployments/biology/image/environment.yml#L47

File: deployments/biology/image/environment.yml [hunk 1]
```diff
@@ -35,5 +35,16 @@ dependencies:
 - raxml=8.2.*
 - muscle=3.8.*
 - dendropy=4.4.*
+
+# requirements.txt
```
Comment (yuvipanda) [inline deployments/biology/image/environment.yml]:
```suggestion
```

Don't think we need to mention this

**Review by yuvipanda — APPROVED** [review]
This LGTM!

**Pull request merged**